Cabal/Distribution/Simple/Command.hs

```diff
@@ -184,16 +191,87 @@ optArg' ad mkflag showflag =
 noArg :: (Eq b) => b -> MkOptDescr (a -> b) (b -> a -> a) a
 noArg flag sf lf d = choiceOpt [(flag, (sf,lf), d)] sf lf d
 
-boolOpt :: (b -> Maybe Bool) -> (Bool -> b) -> SFlags -> SFlags
```

Comment:
OMG, you don't have to do this, but I would really love docs for these functions.